lib/chef/resource/apt_repository.rb

```diff
@@ -164,6 +164,11 @@ class AptRepository < Chef::Resource
       property :key_proxy, [String, nil, FalseClass],
         description: "If set, a specified proxy is passed to GPG via `http-proxy=`."
 
+      # should this default to tru on ubuntu 22.04 and bookworm and  later?
```

Comment:
I think this [Stack Exchange answer](https://askubuntu.com/a/1392143) says support for signed-by was added in Ubuntu 16.04 via apt 1.2, which matches the apt version history in [16.04](https://launchpad.net/ubuntu/xenial/amd64/apt).